rmc/resources/rmc.py

```diff
@@ -83,6 +83,68 @@
 HT is annotated with observed and expected variant counts per base.
 """
 
+CONSTRAINT_ANNOTATIONS = {
+    "mu_snp",
+    "observed",
+    "coverage",
+    "total_exp",
+    "total_mu",
+    "total_obs",
+    "cumulative_obs",
+    "cumulative_exp",
+    "forward_oe",
+    "mu_scan",
+    "section_mu",
+    "section_exp",
+    "section_obs",
+    "section_oe",
+    "reverse",
+    "reverse_obs_exp",
+    "total_null",
+    "total_alt",
+    "chisq",
+    "max_chisq",
+}
```

Comment:
some of these are still unnecessary, right? like `reverse`, `reverse_obs_exp`, `forward_oe`? actually also the `total_exp/mu/obs` shouldn't exist moving forwards

Reply:
To be resolved later!